crates/runner/src/memory/mod.rs

```diff
@@ -555,7 +651,7 @@ mod tests {
 
         memory.insert(addr, value).unwrap();
         assert_eq!(memory.get_data(addr).unwrap(), value.0 .0);
-        assert_eq!(memory.data.len(), 43);
+        assert_eq!(memory.locals.len(), 43);
```

Comment:
below in test_validate_address,         
`assert!(Memory::validate_address((1 << MAX_MEMORY_SIZE_BITS).into()).is_ok());` seems not ok?